src/scss/initial/_links.scss

```diff
@@ -21,14 +21,6 @@
 /// Text underlines are also applied,
 /// but the target depends on the presence or absence of icons,
 /// in order to avoid awkward-looking icon underlines.
-/// @example njk
-///   {% import 'utility.macros.njk' as utility %}
-///   <a href="#">simple text link</a>
-///   {{ utility.icon_link(
-///     icon='icons/external.svg',
-///     text='link with icon',
-///     url='#'
-///   ) }}
```

Comment:
We can't use Nunjucks examples unless we pass in a custom Nunjucks environment that has all our Eleventy filters/shortcodes added. I think we can do that if we want -- it'll require some changes to how we're calling SassDoc/Herman.

Reply:
I think we should for-sure allow adding these docs – the question is if we need to parse & display the results. For now I'd do whichever is simpler:

- Either allow njk examples as plain text without parsing/rendering results. Basically just make it possible to silence the error
- Or pass in the njk env to parse/render results